spec.emu

```diff
@@ -86,7 +86,7 @@ contributors: Google, Ecma International
           </tr>
           <tr>
             <td>*"hebrew"*</td>
-            <td>Hebrew calendar, proleptic. Lunisolar calendar with one leap month inserted after month 5. There is one era.</td>
+            <td>Hebrew calendar, proleptic. Civil calendar with Tishrei the first month of the year. Lunisolar calendar with one leap month inserted after month 5. There is one era.</td>
```

Comment:
A nit again because I think I'm not exactly getting the terseness of this table but I prefer "Tishrei as the/Tishrei being the".